### DISCUSSION TITLE
memes

### DISCUSSION THREAD

**Discussion by esthicodes** [Estheryu991/SwissGerman_Dictionary#51] [General]
stife: eine Berufsausbildung machen
- Stifti: Berufsausbildung
- Stygg: toller Kerl, Bester, Held
- stillständlige: beim Stillstehen
- stober: verdutzt, verwirrt blickend
istoffle, stöffele: tappig gehen, vor allem von kleinen Kindern
- - Stöffeler, Stöffeli: Kleinkind, eben laufen gelernt
- Stogel: festgetretener Schneeklumpen an den Schuhen
- Stögele: auf hohen Absätzen trippeln
- Stögeli: hoher Schuhabsatz
- stogle: stolpern, stolpernd gehen
- Stogli: Stolperer, unbeholfener Mensch
- Stöpple: per Anhalter reisen
- Stöppler: Anhalter
- Store: Rollvorhang aus starkem Stoff; Rollladen, Markise, Jalousie
- Storete: grosse Portion
- Storze: Gemüsestrunk, Strunkstengel, (derb/ Bein
- storze: 1. hervorragen, 2. zu Fall bringen,
3. stapfen
- Stöss, Stössli; Hosenbeine, Uberärmel
@cajac

**Comment by esthicodes** [comment]
Wo ich es bebe Doch korrekt gsi, bi han ich gern delfin gha.

@john

**Comment by esthicodes** [comment]
Cervelat
blätze
gleitig. tifig
zugiq
strudlig
Fäcke
Flue
chüschele
nachega
gfölgig
föpple, helke
Helkerei
fötele
wältsch
Gränne
uschaflech,uschaflig
fryli, frylech
Grageel
Gspane
früüre
Hackblätzli
Coiffeur, Haarschelm, Hörlidieb
Fryse
zytig
Biecht
Zmorge
zmörgele
Füflyber, Grampoolschybe, Schnägg
gförchtig, gfürchtig
schute
Schüteler, Schuter
Alainte

flicken
flink
flott
flüchtig
Flügel
Fluh
flüstern
folgen
folgsam
foppen
Fopperei
fotografieren
französisch
Fratze
frech
freilich
Freudentumult
Freund
frieren
Frikadelle
Frisör
Brisur
Fröstler
frühmorgens
Frühreif
Frühstück
frühstücken
Fünffrankenstück
furchterregend
Fussball spielen
Fussballspieler
Futterbarren

**Comment by muellvin** [comment]
gäl: gelb 
Mini Schueh sind gäl.: Meine Schuhe sind gelb.

\color{yellow}
